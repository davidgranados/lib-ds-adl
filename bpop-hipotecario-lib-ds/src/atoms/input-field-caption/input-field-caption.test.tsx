import React from "react";
import { mount, ReactWrapper } from "enzyme";
import InputFieldCaption from "./input-field-caption";

describe("<InputFieldCaption />", () => {
  let wrapper: ReactWrapper<typeof InputFieldCaption>;
  beforeEach(() => {
    jest.clearAllMocks();
    // given
    wrapper = mount<typeof InputFieldCaption>(<InputFieldCaption>caption</InputFieldCaption>);
  });
  it("should match snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
