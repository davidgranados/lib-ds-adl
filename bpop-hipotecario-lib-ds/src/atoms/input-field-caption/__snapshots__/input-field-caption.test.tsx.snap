// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InputFieldCaption /> should match snapshot 1`] = `
<span
  className="default"
>
  caption
</span>
`;
